mods/cnc/maps/gdi06/rules.yaml

```diff
@@ -73,16 +73,16 @@ SAM:
 RMBO:
 	MustBeDestroyed:
 		RequiredForShortGame: true
-	AutoTarget:
-		TargetWhenIdle: false
-		TargetWhenDamaged: true
 	Health:
 		HP: 150
 
 RMBO.easy:
 	Inherits: RMBO
 	Health:
 		HP: 300
+	AutoTarget:
+		EnableStances: false
```

Comment:
I thought it's not needed there as `RMBO.hard` already removes all instances of `AutoTarget`, but now I see that we use the normal `RMBO` as well. Will fix.